### PR TITLE
Include Rust container definition.

### PR DESCRIPTION
**Pull request by daithiocrualaoich**
14d3762575a

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,6 +13,9 @@
                                               Woof Woof, Inc.
 
 
+[![License](https://img.shields.io/badge/license-Apache--2.0%20OR%20MIT-blue.svg)](https://github.com/woofwoofinc/contributors#license)
```
Comment (passy) [inline README.md]: They are good badges, Dothi.

File: rust/rust.acbuild.sh [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+acbuild run -- rm rustup.sh
+
+acbuild run -- /root/.cargo/bin/cargo install rustfmt
+acbuild run -- /root/.cargo/bin/cargo install cargo-watch
+acbuild run -- /root/.cargo/bin/cargo install cargo-outdated
+acbuild run -- /root/.cargo/bin/cargo install cargo-graph
+acbuild run -- /root/.cargo/bin/cargo install cargo-modules
+acbuild run -- /root/.cargo/bin/cargo install cargo-count
+
+acbuild run -- /root/.cargo/bin/rustup install nightly
```
Comment (passy) [inline rust/rust.acbuild.sh]: Is that necessary if we can't build clippy?

File: rust/rust.acbuild.sh [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+acbuild end
+
+
+################################################################################
+# Cleanup
+################################################################################
+
+popd > /dev/null
+rm -fr rust
```
Comment (passy) [inline rust/rust.acbuild.sh]: `-fr`? Sorry, but that's just wrong. How do you even pronounce that? For "rimraf" it's clear ... but this? "rimfra"? "rimfar"?

File: rust/rust.acbuild.sh (hunk 2 above)
Comment (daithiocrualaoich) [inline rust/rust.acbuild.sh]: No but it represents an aspiration that clippy will be restored eventually.

File: rust/rust.acbuild.sh (hunk 3 above)
Comment (daithiocrualaoich) [inline rust/rust.acbuild.sh]: Although, this script is likely to accidentally hose anything named rust. It's on my backlog to tmp-name the build directory in these scripts. Can't help with your spiritual confusion around -fr/-rf... :)

**Pull request merged**